pkg/controller/storagecluster/reconcile.go

```diff
@@ -502,6 +503,29 @@ func (r *ReconcileStorageCluster) setRookCleanupPolicy(instance *ocsv1.StorageCl
 	return nil
 }
 
+// set noobaa cleanup policy to destroy obc
+func (r *ReconcileStorageCluster) setNoobaaCleanupPolicy(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```

Comment:
Please have "Uninstall:" as a prefix for all the log lines.

Also, have a log line that prints a success case, may be at line 527.

Reply:
Done